app/views/content_items/detailed_guide.html.erb

```diff
@@ -3,6 +3,7 @@
     schema: :faq
   ) %>
 <% end %>
+<%= render 'shared/intervention_banner' %>
```

Comment:
Should this be in a guard clause? So it doesn't need to be added and removed everytime?
e.g. https://github.com/alphagov/frontend/blob/735e8c9e1d5bcab90994651009437e9127e37b58/app/views/shared/_intervention_banner.html.erb#L1

Reply:
I think the field values should be added too?

Reply:
Sorry, I missed that it's in it's own partial 🤦‍♀️